__manifest__.py

```diff
@@ -14,6 +14,7 @@
         "views/openacademy_menu.xml",
         "views/openacademy_course_views.xml",
         "views/openacademy_session_views.xml",
+        "report/openacademy_session_report.xml",
         "views/res_partner_views.xml",
```

Comment:
I think it is a good practice to have the files of the same folder in sequence rather than organizing them by models.

```suggestion
        "views/openacademy_menu.xml",
        "views/openacademy_course_views.xml",
        "views/openacademy_session_views.xml",
        "views/res_partner_views.xml",
        "report/openacademy_session_report.xml",
```

Reply:
Got it.